src/EventHandler/Message/GroupMessage.php

```diff
@@ -60,6 +66,80 @@ public function getMember(string|int|null $member = null): Participant
         };
     }
 
+    /**
+     * Display the participants list in a [supergroup](https://core.telegram.org/api/channel).
+     * The supergroup must have at least `hidden_members_group_size_min` participants in order to use this method, as specified by the [client configuration parameters »](https://core.telegram.org/api/config#client-configuration).
+     *
+     * @return void
+     * @throws InvalidArgumentException
+     */
+    public function showMembers(): void
```

Comment:
`unhideMembers` is maybe better, showMembers somehow reminds me of getMembers, which is not what this function does